docs/application/native/guides/text-input/tts.md

```diff
@@ -686,6 +719,32 @@ To start, pause, and stop the playback, follow the steps below:
   }
   ```
 
+<a name="control"></a>
+## Playing Mode
+
+It is possible to determine whether to request playback of PCM data synthesized by the TTS service or to receive and play it directly by the TTS client.
```

Comment:
```suggestion
Playing mode determines which process, either TTS service or TTS client, plays the synthesized PCM data. There are 2 different playing modes. They are described below:
- `TTS_PLAYING_MODE_BY_SERVICE`: TTS service plays the synthesized PCM data. If you do not set playing mode, it will be set as `TTS_PLAYING_MODE_BY_SERVICE`.
- `TTS_PLAYING_MODE_BY_CLIENT`: TTS client receives the synthesized PCM data from TTS service, and plays it directly.
```

Reply:
Thank you for changing this more properly. I've changed this.